instance/tests/integration/integration_instance.py

```diff
@@ -164,15 +164,14 @@ def assert_mongo_db_provisioned(self, instance):
             db.logout()
 
     @shard(1)
-    def test_activate_multiple_appservers(self):
```

Comment:
Had to remove the multi-vm integration tests, because [CircleCI was timing out](https://circleci.com/gh/open-craft/opencraft/2050?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link)